resolver/dns_resolver.ml

```diff
@@ -8,6 +8,9 @@ type key = [ `raw ] Domain_name.t * Packet.Question.qtype
 
 let pp_key = Dns_resolver_cache.pp_question
 
+let src = Logs.Src.create "dns-resolver" ~doc:"DNS resolver"
```

Comment:
```suggestion
let src = Logs.Src.create "dns_resolver" ~doc:"DNS resolver"
```

let's stay consistent across this repository.

Reply:
Yes of course, sorry for that.